src/modules/search/components/ClientSearch.tsx

```diff
@@ -205,6 +211,8 @@ const ClientSearch: React.FC<Props> = ({
           disabled={!hasResults}
           cardsEnabled={!!cards}
           onChangeCards={handleChangeDisplayType}
+          sortOrder={sortOrder}
+          onChangeSortOrder={setSortOrder}
```

Comment:
Can we make it so changing the sort order also brings the offset back to 0?

Reply:
Fixed